### PR TITLE
mgr/dashboard: run backend api tests

### PR DESCRIPTION
**Pull request by alfonsomthd**
Disable coverage by default as this info is not checked
anymore and it is causing an error in environment:
docker centos7 + python 2.7

Signed-off-by: Alfonso Martínez <almartin@redhat.com>

### CODE REVIEW

**Comment by LenzGr** [comment]
I think the title of this PR and the commit message is misleading - can you please rephrase this to be more explicit about what this change actually does?

Wouldn't "disabled backend coverage tests" be more appropriate? And if this is the case, I agree with @rjfd  that it would be nice to keep track of re-enabling them at some point again. Could you please submit a tracker issue about this?

Thank you!

**Comment by alfonsomthd** [comment]
@LenzGr @rjfd Updated title and created the related issue:
https://tracker.ceph.com/issues/36176

**Comment by ricardoasmarques** [comment]
Jenkins retest this

**Comment by ricardoasmarques** [comment]
jenkins retest this please

**Comment by callithea** [comment]
jenkins retest this please
